pkg/metrics/metrics.go

```diff
@@ -0,0 +1,208 @@
+// Package metrics provides functionality for recording and serving
+// metric information related to AWS API calls and internal driver operations.
+package metrics
+
+import (
+	"net/http"
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/klog/v2"
+)
+
+const (
+	// Metric types
+	AWS metricType = iota
+	Internal
```

Comment:
np: This enum should be in its own `const` block.

Reply:
Fixed ✅